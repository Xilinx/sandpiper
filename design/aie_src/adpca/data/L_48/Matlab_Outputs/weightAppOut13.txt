-1.530488586e+01 9.213199615e+00
-3.325982094e+00 3.833889961e+00
2.643445015e+00 6.692464828e+00
