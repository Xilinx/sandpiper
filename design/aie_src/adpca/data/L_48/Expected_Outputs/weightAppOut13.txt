-1.530437279e+01 9.212556839e+00 
-3.325977325e+00 3.833745718e+00 
2.643419743e+00 6.692207336e+00 
